src/builds.ts

```diff
@@ -98,6 +98,30 @@ export interface ProjectBuildOptions {
     /** Use babel to compile all ES6 JS down to ES5 for older browsers. */
     compile?: boolean
   };
+
+  /**
+   * Browser capabilities required for serving this build. Used by servers that
+   * support differential serving to decide which build to serve to a given
+   * user agent.
+   *
+   * Values include `es2015` and `push`. See canonical list at:
+   * https://github.com/Polymer/prpl-server-node/blob/master/src/capabilities.ts
+   */
+  capabilities?: string[];
```

Comment:
or `browser: { capabilities: ... }` or `browserCapabilities`

One other nice thing about having a whole stanza for `client` or `browser` is that if we add other hooks for differential serving they'd be grouped together making it easier to reason about.  I could also see a scenario where someone wanted to specifically differentially serve based on something they knew about the browser that wasn't entirely manageable by our capabilities model.  These would be things like serve-this-one-for-the-facebook post crawler thing etc.